.docker/public-nginx.conf

```diff
@@ -26,13 +26,12 @@ server {
 server {
     server_name ~^[a-z]+\.libero\.pub$;
     listen 80;
+    root /app/public;
```

Comment:
`/app/browser/public`, or even `/app/browser`?

Reply:
(dictated by the original image at the moment)